account_payment_return/models/payment_return.py

```diff
@@ -251,7 +252,10 @@ def action_confirm(self):
                 )
             extra_lines_vals = return_line._prepare_extra_move_lines(move)
             move_line_model.create(extra_lines_vals)
+            debit_move_partner = move_line2.partner_id
```

Comment:
What's happen if we have several return lines with different partner?

Reply:
Right now it uses the partner of the last return line, which in hindsight is questionable, but since this is 4 months old and as far as I know there haven't been any issues with it, is having different partners a real use case? I'm not really educated on these kind of accounting things :sweat_smile: